blog/2022-09-20-consensus.md

```diff
@@ -1,3 +1,11 @@
+---
+title: Consensus Team Update
+slug: 2022-09-20-consensus
+authors: dnadales
```

Comment:
Do you know how where the mapping `dnadales` to my name and role is defined?

Reply:
Nevermind, I was doing a linear scan of the PR. Would it be worth mentioning the `authors` file in the README?